src/hearings/models/hearings.enum.ts

```diff
@@ -97,6 +97,7 @@ export enum HearingCategory {
   NonStdDurationReasonCodes = 'NonStdDurationReasonCodes',
   EntityRoleCodes = 'EntityRoleCodes',
   JudgeType = 'JudgeType',
+  ListOffValues = 'ListOffValues',
```

Comment:
This LoV is a generic method, it should be category specific, so I think it should be something like 'OtherPanelRoles'